spec/spec_helper.rb

```diff
@@ -18,7 +18,7 @@
 #
 
 $TESTING = true
-$:.push File.join(File.dirname(__FILE__), "..", "lib")
+$:.push File.join(__dir__, "..", "lib")
```

Comment:
`lib/` is automatically added to the load path in rspec so this is likely unnecessary.

Reply:
@phiggins can you make that a suggestion?